docs/how-to/custom-app.md

```diff
@@ -0,0 +1,17 @@
+# Define a custom app ID
+
+To create a new AnyCloud app with a custom name simple pass the optional `-a`, or `--app-id` parameter
+
+```
+$ anycloud new staging -a test
```

Comment:
People may come into this page directly, not having read the Node express hello world tutorial, so it would probably be good to explain what `staging` is referring to.

Reply:
yes, thanks!